packages/repository/src/model.ts

```diff
@@ -145,6 +149,63 @@ export class ModelDefinition {
     return this;
   }
 
+  /**
```

Comment:
What't the intention of adding these helpers? And I think relations should not be the part of the model definition.